test/e2e/harness.ts

```diff
@@ -0,0 +1,34 @@
+import { Page } from "@recordreplay/playwright";
+
+declare global {
+  const app: any;
```

Comment:
If we have types for app state and app actions/selectors (which I think we have at least *some* of), it might be nice to give this just a slightly more deliberate type. Eh, on second thought, I don't know if it's worth it. I leave it up to you and other reviewers to decide if there is any value there.

Reply:
We have an `AppHelpers` interface for that, I tried that but it's incomplete and didn't contain the methods I needed so I postponed this.

Reply:
It looks like we are at least implicitly defining this as

```
app: {
  actions: any;
  selectors: any;
} | undefined;
```

So that could be worth being explicit about, but not a huge deal.